src/components/cc-pricing-product-consumption/cc-pricing-product-consumption.js

```diff
@@ -89,16 +88,10 @@ export class CcPricingProductConsumption extends withResizeObserver(LitElement)
 
     this._simulator = new PricingConsumptionSimulator();
 
-    /** @type {number|null} Set by the `withResizeObserver` mixin. The width of the component in `px`. See the `onResize` method for more info. */
-    this._size = null;
-
     /** @type {SectionStates} Sets the state of the section. It is modified everytime the quantity is changed, the section is shown / hidden, or if the unit value changes. */
     this._sectionStates = {};
-  }
 
-  /* Used by the `withResizeObserver` mixin. */
-  onResize ({ width }) {
-    this._size = width;
+    new ResizeController(this);
```

Comment:
issue: let's keep a reference to the controller so we can get its `width` property

```js
this._resizeController = new ResizeController(this);
```